lib/metasploit/framework/login_scanner/teamcity.rb

```diff
@@ -0,0 +1,181 @@
+require 'metasploit/framework/login_scanner/http'
+require 'rex/proto/teamcity/rsa'
+
+module Metasploit
+  module Framework
+    module LoginScanner
+
+      # This is the LoginScanner class for dealing with JetBrains TeamCity instances.
+      # It is responsible for taking a single target, and a list of credentials
+      # and attempting them. It then saves the results.
+      class Teamcity < HTTP
+        DEFAULT_PORT         = 8111
+        LIKELY_PORTS         = [8111]
+        LIKELY_SERVICE_NAMES = ['skynetflow'] # Comes from nmap 7.95 on MacOS
```

Comment:
I think this is just a mismatch of scanning port 8111:

https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=8111